_posts/2019/2019-04-14-prevent-links-in-text-fields-to-foil-spammers.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Prevent Links in Text Fields to Foil Spammers"
+description: "Bad guys on the internet are inventive."
+layout: article
+category: ruby
+image:
+  base: "2019/prevent-links-in-text-fields-to-foil-spammers"
+  alt: "SPAM. Spam. Lovely Spam."
+  source: "https://www.flickr.com/photos/jeepersmedia/14014120505/"
+  credit: "Mike Mozart"
+---
+
+Your application most likely sends email even if that is not a primary function of the product: invites, messages & forgot password reminders.
+
+As soon as you allow user generated content in those emails your application becomes an interesting target for email spammers.
```

Comment:
Comma after "in those emails"

Reply:
Maybe you need an example of what you mean by 'user generated content' — it could just be something in parenthesis or in between em-dashes.

Reply:
*user-generated